containers/Investment/RevolvingPool/View/PoolOverview.tsx

```diff
@@ -1,53 +1,192 @@
+              {addThousandsSeparators(toPrecision(baseToDisplay(poolValue, 18), 2))} DAI
+            </Heading>
+          </Box>
+
+          <Table>
+            <TableBody>
+              <TableRow>
+                <TableCell scope="row">Current NAV</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(toPrecision(baseToDisplay((pool.data as PoolDataV3).netAssetValue, 18), 2))}{' '}
+                  DAI
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell scope="row">Current Reserve</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(toPrecision(baseToDisplay((pool.data as PoolDataV3).reserve, 18), 2))} DAI
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell scope="row">Maximum Reserve Amount</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(toPrecision(baseToDisplay((pool.data as PoolDataV3).maxReserve, 18), 2))} DAI
+                </TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
 
-      <Table>
-        <TableBody>
-          <TableRow>
-            <TableCell scope="row">Current NAV</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 20,500</TableCell>
-          </TableRow>
-          <TableRow>
-            <TableCell scope="row">Current Reserve</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 20,500</TableCell>
-          </TableRow>
-        </TableBody>
-      </Table>
-
-      <Heading level="5" margin={{ bottom: 'small' }}>
-        Assets
-      </Heading>
-      <Table>
-        <TableBody>
-          <TableRow>
-            <TableCell scope="row">Outstanding Volume</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>DAI 1,000.510</TableCell>
-          </TableRow>
-          <TableRow>
-            <TableCell scope="row">DROP APR</TableCell>
-            <TableCell style={{ textAlign: 'end' }}>10.50%</TableCell>
-          </TableRow>
-        </TableBody>
-      </Table>
-    </Box>
+          <Heading level="5" margin={{ bottom: 'small' }}>
+            Assets
+          </Heading>
+          <Table>
+            <TableBody>
+              <TableRow>
+                <TableCell scope="row">Outstanding Volume</TableCell>
+                <TableCell style={{ textAlign: 'end' }}>
+                  {addThousandsSeparators(
+                    toPrecision(baseToDisplay((pool.data as PoolDataV3).outstandingVolume, 18), 2)
+                  )}{' '}
+                  DAI
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell scope="row">
+                  <span>
+                    <TokenLogo src={`../../../../static/DROP_final.svg`} />
```

Comment:
Can we use absolute paths for static imports? 
```suggestion
                    <TokenLogo src="/static/DROP_final.svg" />
```